efcms-service/serverless.yml

```diff
@@ -76,8 +76,8 @@ provider:
   stage: ${self:custom.stage}
   region: ${self:custom.region}
   runtime: nodejs8.10
-  memorySize: 128
-  timeout: 10
+  memorySize: 768
```

Comment:
Here's a good article on the topic: https://epsagon.com/blog/how-to-make-lambda-faster-memory-performance-benchmark/

As we get closer to production, we'll want to spend some time to revisit the memory sweet spot.

Reply:
I'll read that over — thank you!

Reply:
The lower the memory, the less CPU allocation your lambda gets.  We increased this because we were seeing very slow execution times (2-3 seconds) for a simple dynamodb query.  Bumping this to 768MB allows for enough CPU to get that same request reduced to around 200ms.  It ends up being cheaper to use the larger memory.

```
COSTS:
128MB - 3000ms / 100ms = 30 * $0.000000208 = $0.00000624
768MB - 300ms  / 100ms =  3 * $0.000001250    = $0.00000375
```

Reply:
That's so interesting!